### DISCUSSION TITLE
Multiple panels at the same time?

### DISCUSSION THREAD

**Discussion by andrejohansson** [zed-industries/zed#18378] [Feature Requests]
Is it possible to have multple panels open at the same time somehow? Eg. I would want to have both the Project Panel and the Strucuture panel open at the same time taking up half the height each.

**Comment by SomeoneToIgnore** [comment]
![image](https://github.com/user-attachments/assets/f1b57fa9-f846-4dab-bfc8-b62236d8bb7c)

If you move them to other docs (options are: left/right/bottom yet not every panel allows to be moved everywhere e.g. project/outline ones do not move to the bottom), it's possible.

**Reply by andrejohansson** [reply]
Ahh, right click on their icon in the bottom bar and click dock right/left. Thank you!

**Comment by travisgriggs** [comment]
This works as fine as you're not showing the Agent panel... I would *love* if I could have two left side panels, file then outline.

**Reply by peterlionelnewman** [reply]
up vote for 'top-left, bottom-left, middle-left, etc' paneling -> @JosephTLyons not sure if this is already a feature request somewhere?

**Comment by vhp1360** [comment]
Hello everyone.
this is very primitive features.
Project,outline and git panel together in left for example . multiple panel at the same time.
it is very strange that a powerful IDE like Zed doesnt have this feature.
thank you so much

**Comment by vanridal** [comment]
Agreed, it leaves a lot of empty real-estate on the screen not having split panels

**Comment by dcrdev** [comment]
Just switched to zed and this would be great to have
